specification/specs/x_pack/eql/EqlHits.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+class EqlHits<TEvent> {
+  total?: TotalHits
+  events?: EqlHitsEvent<TEvent>[]
+  sequences?: EqlHitsSequence<TEvent>[]
```

Comment:
I think these properties are always present, so there is no need to add the `?`.

Reply:
`sequences` are conditional to the `sequence` keyword in a EQL query and if `sequences` is an available property, the `events` is not 😄 
This tests shows that `total` is also conditional: `xpack-Execute_EQL_events_query_with_filter_path_2_response.json.test-d.ts`

In other words all 3 properties must stay optional.